app/views/samples/index.html.erb

```diff
@@ -25,5 +25,5 @@
       </script>
   <% end %>
 <% else %>
-    <%= render partial: "general/index" %>
+    <%= render partial: "general/index",:locals=>{:show_query_button=>true} %>
```

Comment:
as mentioned, instead of always passing true, it should pass `isa_templates_enabled`